### ISSUE TITLE
Large number of kernel log lines through bpf_probe_write_user helper

### ISSUE THREAD

**Issue by thatsdone** [open-telemetry/opentelemetry-go-instrumentation#252]
### Describe the bug

When I tried opentelemetry-go-instrumentation, I saw bunch of kernel log (warning?) messages like below.
I think it's better to suppress even if this is normal behavior  from a view point of operation because amount of the  lines is huge.

```
$ sudo dmsg
(snip)
[481254.185220] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.185377] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.186084] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.186234] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.186284] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.186301] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.188186] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.189146] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.191409] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481254.191678] otel-go-instrum[42595] is installing a program with bpf_probe_write_user helper that may corrupt user memory!
[481259.458121] bpf_get_probe_write_proto: 38 callbacks suppressed
(snip)
```

### Environment

 - OS: Ubuntu 22.04.1 LTS (amd64), kernel 5.15.0-52-generic
 - Go Version: 1.20 (I think, I'm using release binaries)
 - Version: v0.2.2-alpha  (dockerhub image digest: sha256:03448c64be10365fa2fa1078ec2aed12ee1d6c91aea7e70beadf506bb3d5fc4c)
 - kubernetes; 1.27.4
 - CNI: cilium 1.14.0
 
### To Reproduce

I build my k8s cluster using above versions of components, and then followed the getting started page.
Basically, same as the reproduce procedure with  #251.

### Expected behavior

We don't see the (bunch of) warning (just normal log?) messages above.

### Additional context

* In case of cilium, it looks like there was a similar discussion.
  * https://github.com/cilium/cilium/issues/10048
